RELEASE_NOTES.md

```diff
@@ -2,7 +2,194 @@
+     }
+]
+```
+
+Possible options for each element of the ``api_keys`` list:
+
+| Field   | required | Description                                                                                                                                            |
+|---------|----------|--------------------------------------------------------------------------------------------------------------------------------------------------------|
+| api_key | yes      | The API key token.                                                                                                                                     |
+| id      | yes      | The identifier of the API key. Must be unique. String.                                                                                                 |
+| workers | no       | The number of [workers](#multi-worker-configuration) for the API key. By default it has the same value as the `default_workers_per_api_key` [option](#multi-worker-configuration). |
+
+You may also split the definition of the api_keys field across [multiple configuration files](https://app.scalyr.com/help/scalyr-agent?teamToken=7bn1PMhjS%2F8335sB6vgDcQ--#modularConfig)
+in the agent.d directory. The agent will join together all entries in all api_keys fields defined.
+
+NOTE: The "main" API key, which is defined in the ``api_key`` section, also has it's own implicit identifier - ``"default"``.
+
+The identifier ``"default"`` is reserved by the "main" API key and can not be used by other API keys. For example, the next configuration is invalid.
+```
+{
+    "api_key": "<main_key>",
+    "api_keys": [
+         {
+             "api_key": "<second_key>",
+             "id": "default",  // WRONG, the <main_key> API key already has the identifier "default".
+         },
+    ]
+}
+```
+
+As an exception, the ``"default"`` identifier can be used with the same ``api_key`` value as in the "main" ``api_key`` section to change default options for this API key.
+
+```
+{
+    "api_key": "<main_key>",
+    "api_keys": [
+         {
+             "api_key": "<main_key>", // the same API key as in the "api_key" section.
+             "id": "default",
+             "workers": 3 // change the workers number from default - 1 to 3.
+         },
+         {
+             "api_key": "<second_key>",
+             "id": "second",
+         },
+    ]
+}
+```
+
+#### Associating logs with API keys.
+
+Each element of the ``logs`` section can be associated with a particular API key by specifying the ``api_key_id`` field.
+
+NOTE: If the ``api_key_id`` is omitted, then the ``default`` API key is used.
+```
+"logs": [
+    {
+       "path": "/var/log/frontend/*.log",
+       "api_key_id": "frontend_team" // the API key with identifier - "frontend_team" is used.
+    }
+    {
+       "path": "/var/log/backend/*.log"
+       // the "api_key_id" is omitted, the API key with "default" identifier is used.
+    }
+]
+```
+
+
+##### Full configuration example.
```

Comment:
Let's remove this "full configuration example".  I think it's a little too much for the release notes.